### ISSUE TITLE
Splitter: Keyboard accessibility controls

### ISSUE THREAD

**Issue by 08screasey** [ant-design/ant-design#55930]
### What problem does this feature solve?

The Splitter pane is not currently accessible for users who navigate with a keyboard. When testing the Splitter on the ant examples page there are no focusable elements within the Splitter bar dragger. By adding a button element that a user can focus and click with the keyboard, users who cannot otherwise drag the bar with a mouse will be able to resize the panes.

### What does the proposed API look like?

- Adjust the splitter bar dragger to be a focusable element, which displays highlighted/active styles when the user presses 'Enter' or 'Space'.
- Once active, a keyboard listener is added and listens to "ArrowDown", "ArrowUp" or "ArrowLeft", "ArrowRight" depending on the orientation. On each key press the pane is moved a specific amount in that direction, until reaching the min/max. When "Shift" is pressed, the size of each step is increased.
- No additional props are required to be exposed to the user, although a `keyboardStep` prop would be a nice-to-have, for developer to be able to customise the step size.

**Comment by afc163** [comment]
感谢您的反馈！这是一个重要的无障碍功能请求。Splitter 组件目前缺少键盘控制支持，这对于无障碍访问非常重要。

我们需要支持以下键盘操作：
- 方向键调整分隔面板大小
- Enter/Space 开始和结束调整

欢迎社区贡献此功能！
